docs/testing/asserts.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Asserts
+type: docs
+menu:
+  docs:
+    parent: 'Testing'
+weight: 4
+---
+
+# Asserts
+
+Test asserts are the part of a [test step](/docs/testing/steps) that define the state to wait for Kubernetes to reach. It is possible to match specific objects by name as well as match any object that matches a defined state.
+
+* [Format](#format)
+* [Getting a resource from the cluster](#getting-a-resource-from-the-cluster)
+* [Listing resources in the cluster](#listing-resources-in-the-cluster)
+
+# Format
```

Comment:
shouldn't this be ## ? as a sub-section

Reply:
Incidentally, the only reason I included a title at the top is our Hugo style omits the page title. Should I maybe fix the style and drop the titles at the top of the page?